### PR TITLE
M2-6130: Number value is changing after scrolling

### PR DESCRIPTION
**Pull request by nmialik**
- [ ] Delivered the `fix` or `feature` branches into `develop` or `release` branches via `Squash and Merge` (to keep clean history)

### 📝 Description

🔗 [Jira Ticket M2-6130](https://mindlogger.atlassian.net/browse/M2-6130)

### CODE REVIEW

File: src/shared/components/FormComponents/InputController/InputController.tsx [hunk 1]
```diff
@@ -10,20 +11,34 @@ export const InputController = <T extends FieldValues>({
...
-        onCustomChange={onCustomChange}
-        {...props}
-      />
-    )}
-  />
-);
+}: InputControllerProps<T>) => {
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  const handleOnWheel = () => {
```
Comment (IhorNesterenko99) [inline src/shared/components/FormComponents/InputController/InputController.tsx:20]: Could you please leave the comment, why this change is needed?

**Review by alex-i-e — APPROVED** [review]
